sdk/swap-service/tests/oneinch.spec.ts

```diff
@@ -3,8 +3,8 @@ import { Address, Percentage, Token, TokenAmount } from '@summerfi/sdk-common/co
 import { subtractPercentage } from '@summerfi/sdk-common/utils'
 import { getSwapManager, SwapData, SwapProviderType } from '~swap-service'
 
-describe.skip('OneInch | SwapManager | Integration', () => {
-  it('should provide swap data', async () => {
+describe('OneInch | SwapManager | Integration', () => {
+  it.skip('should provide swap data', async () => {
```

Comment:
I think Jacob also disabled this on `dev`. Just in case it conflicts when you merge